plugins/overrides/override/override.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension version="1.0" type="plugin" group="overrides" method="upgrade">
+	<name>plg_overrides_override</name>
```

Comment:
I am not sure if the version number is correct. In other plugins the version number of Joomla is used. And for the date only the month and the year are inserted.

```
<extension version="4.0.0"
...
<version>4.0.0.</version>
<creationDate>June 2016</creationDate>
```

Reply:
@laoneo